django/cantusdb_project/main_app/tests/mock_cantusindex_data.py

```diff
@@ -0,0 +1,644 @@
+################################################################################
+
+mock_json_cid_909030_text: str = """
+{
+    "info": {
+        "field_ah_item": null,
+        "field_ah_volume": null,
+        "field_cao": null,
+        "field_cao_concordances": null,
+        "field_feast": "Invitatoria",
+        "field_full_text": "Venite exsultemus domino jubilemus deo salutari nostro praeoccupemus faciem ejus in confessione et in psalmis jubilemus ei | Quoniam deus magnus dominus et rex magnus super omnes deos quoniam non repellet dominus plebem suam quia in manu ejus sunt omnes fines terrae et altitudines montium ipse conspicit | Quoniam ipsius est mare et ipse fecit illud et aridam fundaverunt manus ejus venite adoremus et procidamus ante deum ploremus coram domino qui fecit nos quia ipse est dominus deus noster nos autem populus ejus et oves pascuae ejus | Hodie si vocem ejus audieritis nolite obdurare corda vestra sicut in exacerbatione secundum diem tentationis in deserto ubi tentaverunt me patres vestri probaverunt et viderunt opera mea | Quadraginta annis proximus fui generationi huic et dixi semper hi errant corde ipsi vero non cognoverunt vias meas quibus juravi in ira mea si introibunt in requiem meam | Gloria patri et filio et spiritui sancto sicut erat in principio et nunc et semper et in saecula saeculorum amen",
+        "field_fulltext_source": null,
+        "field_genre": "IP",
+        "field_language": null,
+        "field_notes": null,
+        "field_related_chant_id": null,
+        "field_similar_chant_id": null,
+        "field_troped_chant_id": null
+    },
+    "chants": {
+        "0": {
+            "siglum": "P-Cug MM 0045",
+            "srclink": "https://pemdatabase.eu/source/46513",
+            "chantlink": "https://pemdatabase.eu/musical-item/50730",
+            "folio": "029r",
+            "sequence": null,
+            "incipit": "Venite exsultemus domino jubilemus deo salutari",
+            "feast": "De festis duplicibus",
+            "office": "M",
+            "genre": "IP",
+            "position": "",
+            "image": "https://pemdatabase.eu/image/84007",
+            "mode": "",
+            "fulltext": "Venite exsultemus domino jubilemus deo salutari nostro praeoccupemus faciem ejus in confessione et in psalmis jubilemus ei | Quoniam deus magnus | Quoniam ipsius est mare et ipse fecit illud et aridam fundaverunt manus ejus venite adoremus et procidamus ante deum ploremus coram domino qui fecit nos quia ipse est dominus deus noster nos autem populus ejus et oves pascuae ejus | Hodie si vocem | Quadraginta annis proximus fui generationi huic et dixi semper hi errant corde ipsi vero non cognoverunt vias meas quibus juravi in ira mea si introibunt in requiem meam | Gloria",
+            "melody": "",
+            "db": "PEM"
+		}
+    }
+}
+"""
+mock_json_cid_909030_content: bytes = bytes(
+    mock_json_cid_909030_text,
+    encoding="utf-8-sig",
+)
+mock_json_cid_909030_json: dict = {
```

Comment:
Is this the same as what is in `_content` and `_bytes`? Showing the conversion to json would make more sense to me.